tests/validation/tests/v3_api/test_import_k3s_cluster.py

```diff
@@ -168,7 +169,11 @@ def create_multiple_control_cluster():
                               'cluster_type': RANCHER_CLUSTER_TYPE,
                               'volume_size': AWS_VOLUME_SIZE,
                               'create_lb': str(K3S_CREATE_LB).lower(),
+<<<<<<< HEAD
```

Comment:
some merge conflict artifacts snuck in

Reply:
Ahh thank you, I was lazy and didn't spot this. Fixed!